docker_mounts.go

```diff
@@ -108,7 +108,7 @@ func mapToDockerMounts(containerMounts ContainerMounts) []mount.Mount {
 		case TmpfsMounter:
 			containerMount.TmpfsOptions = typedMounter.GetTmpfsOptions()
 		default:
-			Logger.Printf("Mount type %s is not supported by Testcontainers for Go", m.Source.Type())
+			// The provided source type has no custom options
```

Comment:
Do you think we can add a case for the `BindMounter` with a deprecation logging, and then include your default case?

Reply:
That's a good idea! Done